<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
        <bold>true</bold>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>ðŸ“š Library Management System ðŸ“š</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="Books">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Book">
       <attribute name="title">
        <string>Books</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>ðŸ“š Add Books</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>bookID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_bookID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_title"/>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>author</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_author"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>year</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_year"/>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_category"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addBook">
          <property name="text">
           <string>addBook</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_viewBooks">
          <property name="text">
           <string>viewBooks</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Search Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_searchBook"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_searchBook">
          <property name="text">
           <string>searchBook</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Filter Book</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_filterBook"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_filterBook">
          <property name="text">
           <string>filterBook</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Articles</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_18">
          <property name="text">
           <string>ðŸ“„ Articles</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_19">
          <property name="text">
           <string>articleID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_articleID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_20">
          <property name="text">
           <string>articleTitle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_articleTitle"/>
        </item>
        <item>
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>articleAuthor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_articleAuthor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>articleYear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_articleYear"/>
        </item>
        <item>
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>doi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_doi"/>
        </item>
        <item>
         <widget class="QLabel" name="label_24">
          <property name="text">
           <string>journal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_journal"/>
        </item>
        <item>
         <widget class="QLabel" name="label_25">
          <property name="text">
           <string>articleCategory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_articleCategory"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addArticle">
          <property name="text">
           <string>addArticle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_viewArticles">
          <property name="text">
           <string>viewArticles</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Theses</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="text">
           <string>ðŸŽ“ Theses</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_27">
          <property name="text">
           <string>thesisID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_thesisID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_28">
          <property name="text">
           <string>thesisTitle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_thesisTitle"/>
        </item>
        <item>
         <widget class="QLabel" name="label_29">
          <property name="text">
           <string>thesisAuthor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_thesisAuthor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_30">
          <property name="text">
           <string>thesisYear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_thesisYear"/>
        </item>
        <item>
         <widget class="QLabel" name="label_31">
          <property name="text">
           <string>university</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_university"/>
        </item>
        <item>
         <widget class="QLabel" name="label_32">
          <property name="text">
           <string>degree</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_degree"/>
        </item>
        <item>
         <widget class="QLabel" name="label_33">
          <property name="text">
           <string>advisor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_advisor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_34">
          <property name="text">
           <string>category</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_thesisCategory"/>
        </item>
        <item>
         <widget class="QLabel" name="label_35">
          <property name="text">
           <string>keywords</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_keywords"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addThesis">
          <property name="text">
           <string>addThesis</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_viewTheses">
          <property name="text">
           <string>viewTheses</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Digital Content</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLabel" name="label_36">
          <property name="text">
           <string>ðŸ’¾ Digital Content</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_44">
          <property name="text">
           <string>digitalID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_digitalID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_43">
          <property name="text">
           <string>digitalTitle</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_digitalTitle"/>
        </item>
        <item>
         <widget class="QLabel" name="label_42">
          <property name="text">
           <string>digitalAuthor</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_digitalAuthor"/>
        </item>
        <item>
         <widget class="QLabel" name="label_41">
          <property name="text">
           <string>digitalYear</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_digitalYear"/>
        </item>
        <item>
         <widget class="QLabel" name="label_40">
          <property name="text">
           <string>format</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_format"/>
        </item>
        <item>
         <widget class="QLabel" name="label_39">
          <property name="text">
           <string>url</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_url"/>
        </item>
        <item>
         <widget class="QLabel" name="label_38">
          <property name="text">
           <string>fileSize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_fileSize"/>
        </item>
        <item>
         <widget class="QLabel" name="label_37">
          <property name="text">
           <string>digitalCategory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_digitalCategory"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_addDigital">
          <property name="text">
           <string>addDigital</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="User">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>ðŸ‘¤ Register User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>userID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_userID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>userName</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_userName"/>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>email</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_email"/>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>limit</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_limit"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_registerUser">
          <property name="text">
           <string>registerUser</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Loans">
       <attribute name="title">
        <string>Loans</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>ðŸ“¦ Borrow/Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>loan User ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_loanUserID"/>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>loan Resource ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_loanResID"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_borrow">
          <property name="text">
           <string>borrow</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_return">
          <property name="text">
           <string>return</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
